Automattic/wp-calypso
# Hide the GitHub deployments card in Hosting Config when the site has a deployment

When a site has a deployment configured in the GitHub deployments section, there's no need to show the card anymore since the user is already familiar with the functionality.

This works by making a query to the code deployments API endpoint that retrieves the number of code deployments configured for the given site. If the site has more than one configuration, then we hide the card.



Related to https://github.com/Automattic/dotcom-forge/issues/6693

## Proposed Changes

* Hide the GitHub banner in the hosting config when the site has deployments configured

## Why are these changes being made?


* The banner is redundant for users who are already using the deployments feature

## Testing Instructions



* Use the live link from the comments
* Configure the GitHub deployments for one AT site
* Go to the Hosting config page (live.link/hosting-config/site-name) and notice that the banner is **not** shown
* Check again for a different AT site (without code deployments enabled) and make sure the banner is still shown
* Check the hosting config page with the flag `layout/dotcom-nav-redesign-v2` enabled and disabled (e.g. `?flags=-layout/dotcom-nav-redesign-v2` - for disabled and `?flags=layout/dotcom-nav-redesign-v2` for enabled)

## Pre-merge Checklist
